src/app/private/scripts/services/adapters/announcementAdapter.js

```diff
@@ -229,13 +236,46 @@ angular.module('playerApp')
             description: annoucement.details.description,
             links: annoucement.links,
             sourceId: annoucement.sourceId,
-            createdBy: annoucement.createdBy,
             target: annoucement.target,
             attachments: annoucement.attachments
           }
         }
         var URL = config.URL.ANNOUNCEMENT.RESEND
         return handleHttpRequest(URL, data, 'POST')
       }
+
+     /**
+       * @method verifyAnnouncementData
+       * @desc Verify announcement data is filled before heading to next step
+       * @memberOf Services.announcementService
+       * @param {string}  stepNumber - step number
+       * @param {object}  announcement - announcement
+       * @returns {boolean} returns boolean
+       * @instance
+       */
+      this.verifyAnnouncementData = function (stepNumber, announcement) {
```

Comment:
the announcement adapter cannot be tied to the steps of the creation flow. please remove all references to step number.